sdk/monitor/azingest/CHANGELOG.md

```diff
@@ -1,16 +1,9 @@
 # Release History
 
-## 0.1.2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 0.1.2 (2024-03-12)
 
 ### Other Changes
-* Updated to the latest version of `azcore`.
-* Enabled spans for distributed tracing.
+* This module is now DEPRECATED. The latest supported version of this module is at github.com/Azure/azure-sdk-for-go/sdk/monitor/ingestion/azlogs
```

Comment:
```suggestion
* This module is now DEPRECATED. The latest supported version of this module is at [github.com/Azure/azure-sdk-for-go/sdk/monitor/ingestion/azlogs](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/monitor/ingestion/azlogs)
```

Reply:
fixed in [6d6bd0c](https://github.com/Azure/azure-sdk-for-go/pull/22578/commits/6d6bd0c245f46a4da2b090ab294c1fa51e0d5e6e)